src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -94,161 +152,1305 @@ static DamageTable D_80B65044 = {
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B62F10.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B62FCC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B631F8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63264.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B632BC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B6330C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B633E8.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63410.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63450.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63474.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B636D0.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B636E4.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63758.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B6382C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B6383C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63854.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63888.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B638AC.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B638D4.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63964.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63980.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63A18.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63AC4.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63C28.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63D0C.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B63D88.s")
-
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/EnBigpo_Update.s")
+// bytes per limb, used in draw func? bit over my head
+static u8 D_80B65078[] = {
+    0xFF, 0x04, 0xFF, 0x00, 0xFF, 0x01, 0xFF, 0x02, 0x05, 0x03, 0x00, 0x00,
+};
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Bigpo/func_80B64190.s")
+// used in limbdraw?
+static Vec3f D_80B65084[] = {
+    { 2000.0f, 4000.0f, 0.0f,},
+    {-1000.0f, 1500.0f, -2000.0f,},
+    {-1000.0f, 1500.0f, 2000.0f,},
+}; 
+
+void EnBigpo_Init(Actor* thisx, GlobalContext* globalCtx2) {
+    GlobalContext* globalCtx = globalCtx2;
+    EnBigpo* this = (EnBigpo*)thisx;
+    EnBigpoFireParticle* firesPtr;
+    s32 i;
+
+    Actor_ProcessInitChain(&this->actor, sInitChain);
+
+    // thisx req to match
+    this->switchFlags = GET_BIGPO_SWITCHFLAGS(thisx);
```

Comment:
Hmm... I dunno, ever since I've started adding those comments I get less "You should use this instead" comments that I have to show dont match later, seems to be working.

Reply:
You know, I don't think these comments are needed in the final result but I will admit it's helps reviews understand the process. I guess from that logic yeah it's fine to have, and we can worry about these comments in later documentation passes